src/schema/notifications.ts

```diff
@@ -478,6 +696,34 @@ export class ZimbraNotifications {
 		});
 	};
 
+	private writeAppointmentsToCache = (
+		appointmentListKey: any,
+		dataToWrite: any
+	) => {
+		const variables = getVariablesFromDataId(appointmentListKey);
+		const appointmentList: any = this.cache.readQuery({
+			query: AppointmentsQuery,
+			variables
```

Comment:
can we have multiple Appointments queries in cache?
if yes then how are we checking updated data belongs to any query or not?